src/drivers/uart/src/i8250.rs

```diff
@@ -0,0 +1,138 @@
+// http://www.ti.com/lit/ds/symlink/pc16550d.pdf
```

Comment:
This link is broken. It'd be really nice to have an up-to-date reference document that you're coding against.

Reply:
broken? Damn. It's from coreboot. I argue with them all the time about not committing links, and here's why. I'll remove it.

Reply:
removed.